languages-and-frameworks/python.html.markerb

```diff
@@ -7,198 +7,278 @@ redirect_from: /docs/getting-started/python/
 
-## _Configure the App for Fly_
+## Configure and Deploy your Fly App
 
-Each Fly application needs a `fly.toml` file to tell the system how we'd like to deploy it. That file can be automatically generated with the `flyctl launch` command. We are going to use one of Fly's builtin deployment configurations for Python.
+To configure and launch the app, run the `fly launch` command and follow the wizard. You can set a name for the app and choose your primary region. You can also choose to launch and attach a Postgresql database and/or a Redis database though we are not using either in this example.
 
 ```cmd
-flyctl launch
+fly launch
 ```
 ```output
-? Select organization: demo (demo)
+Creating app in ../flyio/hello-flask
+Scanning source code
+Detected a Python app
+Using the following build configuration:
+        Builder: paketobuildpacks/builder:base
+? Choose an app name (leave blank to generate one): hello-fly-flask
+? Select Organization: Fly.io (fly-io)
+Some regions require a paid plan (bom, fra, maa).
+See https://fly.io/plans to set up a plan.
+
+? Choose a region for deployment: Amsterdam, Netherlands (ams)
+App will use 'ams' region as primary
+
+Created app 'hello-fly-flask' in organization 'fly-io'
+Admin URL: https://fly.io/apps/hello-fly-flask
+Hostname: hello-fly-flask.fly.dev
+? Would you like to set up a Postgresql database now? No
+? Would you like to set up an Upstash Redis database now? No
 Wrote config file fly.toml
+Validating ../flyio/hello-flask/fly.toml
+Platform: machines
+✓ Configuration is valid
+We have generated a simple Procfile for you. Modify it to fit your needs and run "fly deploy" to deploy your application.
 ```
 
 Organizations are a way of sharing applications between Fly users. When you are asked to select an organization, there should be one with your account name; this is your personal organization. Select that.
 
-One thing to know about the builtin Python builder is that it will automatically copy over the contents of the  directory to the deployable image. This is how you can move static assets such as templates and other files to your application. The other thing to know is that it uses a Procfile to run the application; Procfiles are used on other platforms to deploy Python applications so we keep it simple. The Procfile contains instructions for starting the application. Here's the contents of ours:
+One thing to know about the built-in Python builder is that it will automatically copy over the contents of the directory to the deployable image. This is how you can move static assets such as templates and other files to your application.
+
+The other thing to know is that it uses a `Procfile` to run the application. `Procfiles` are used on other platforms to deploy Python applications so we keep it simple. The Procfile contains instructions for starting the application. Here's the contents of ours:
```

Comment:
The auto-created Procfile has this, which is different from what you show:
```
$ cat Procfile 
# Modify this Procfile to fit your needs
web: gunicorn server:app
```

Is it worth mentioning that one has to edit the Procfile to point to `app:app` instead?

Reply:
@Roadmaster thanks for the suggestion! I'll update this part and mention to update the Procfile in that case. 🙂